---
layout: archive
title: "Research"
permalink: /research/
author_profile: true
---

{% include base_path %}

<b>1. Population genetics and genomics</b><br />
<p style="text-indent: 25px;">Population-genetic theories describe how intra-specific genetic diversity is shaped by evolutionary forces 
such as mutation, selection, and genetic drift. Mutation generates genetic variation, while selection and genetic 
drift then dictate the fate of mutations by fitness-related and stochastic effects, respectively. To study the roles 
of mutation, selection, and genetic drift in evolution, my previous research involved hundreds of <i>Escherichia coli</i> 
populations that have evolved for several hundred days in the lab with rich and complex medium under various bottleneck 
sizes and mutation rates (Ho et al. 2021; Wei & Ho et al. 2022). Throughout the experimental evolution, their fitness, 
mutation rates, and metagenomic sequences were tracked. As a result, the overall fitness improvement over time demonstrates
the adaptation is in action. On the other hand, the populations initially with high mutation rates were found to evolve 
lower mutation rates, highlighting spontaneous mutations are still generally deleterious.</p>
<p style="text-indent: 25px;">In the future, the lab will continue to test population-genetic theories about mutation, selection, 
   and genetic drift using the data collected from these experimentally evolved populations. Molecular engineering 
   will be used to directly verify the phenotypic effects of the derived genotypes that potentially contribute 
   to adaptation. The lab will also keep exploring more derived genotypes and phenotypes that are potentially 
   interesting in these populations, such as those related to biofilm formation, gene expression regulation, 
   cell division, metabolic abilities, and antibiotic resistance.</p>
<br />
<b>2. Historical effects in evolution</b><br /> 
<p style="text-indent: 25px;">As microorganisms often encounter highly heterogeneous environments (spatially or temporally) 
   in nature, the lab is interested in using experimental evolution to study questions about how microorganisms 
   respond to changing environments. In particular, how does the adaptation in one environment hinders or facilitates
   the adaptation in another environment? How often should we expect there is a trade-off or trade-up between the 
   adaptive mechanisms for two different environments?</p>
<p style="text-indent: 25px;">Therefore, the lab plans to use the <i>Escherichia coli</i> populations that have adapted to a rich 
   medium to start new experimental evolution in new environments. The corresponding experimental evolution using the
   ancestral stock of <i>E. coli</i> will be also performed as a control. By comparing their evolved genomic profiles
   and growth phenotypes, how the past adaptation alters the rate or the adaptive mechanism in a new adaptive process
   will be identified.</p>
<br />
<b>3. Plastic and genetic changes of gene expression in microbial evolution</b><br />
<p style="text-indent: 25px;">Gene expression alterations are frequently observed in microbial adaptations to environmental challenges.
   However, the evolutionary direction of plastic (immediate) and genetic (long-term) changes may not be consistent.
   My previous research found that, when organisms encounter novel environments, plastic transcriptomic changes are
   often maladaptive (Ho & Zhang 2019; Ho & Zhang 2020). However, it is possible that there is a different trend when
   the population is moved to a frequently encountered environment instead of a novel environment (Ho & Li et al. 2020).</p> 
<p style="text-indent: 25px;">In the future, the lab will study the role of gene expression changes in microbial adaptations to 
   stressful environments, such as different antibiotics or extreme temperatures. The proposed experiment involves
   the <i>Escherichia coli</i> populations previously adapted to the benign environment (LB broth at 37 Â°C). 
   The original gene expression and the plastic gene expression are measured by RNA-seq in the original and a new 
   stressful environment, respectively. After a period of experimental evolution in a stressful environment, the 
   adaptive gene expression is again obtained by RNA-seq. By comparing original, plastic, and adaptive gene expression
   levels, how frequently plastic responses facilitate the new adaptation in different stressful environments can be
   quantified. The molecular mechanisms of beneficial plastic and genetic responses of gene expression can also be 
   revealed.</p>  
<br />
<b>4. Context dependence of mutational features: rates, spectra, and fitness effects</b><br />
<p style="text-indent: 25px;">DNA mutations are the ultimate source of evolution. DNA mutations are also cellular-level traits 
   reflecting the robustness of replication and repairing pathways. Therefore, studying mutational features not only
   enhances our predictability in organismal evolution but also helps us understand the processes critical for cellular
   fitness. In particular, the rate and spectra of spontaneous mutations may constrain the number of mutations that are
   beneficial in the encountering environment. In addition, the fitness effect of a spontaneous mutation in theory
   dictates the fate of that mutation in the population. These mutational features may be different in different
   environments, and the lack of such knowledge limits our understanding of how organisms respond to different
   environments.</p> 
<p style="text-indent: 25px;">To address this need, the lab plans to estimate the rates, the spectra, and the fitness effects
   of spontaneous mutations in different environments using Escherichia coli as a model. The mutation accumulation
   (MA) experiments in E. coli will be conducted, where the non-lethal spontaneous mutations are sampled in a random
   fashion. The follow-up whole-genome sequencing of a large number of MA lines enables reliable measurements of
   mutation rates and spectra of spontaneous mutations. Fitness estimates of these MA lines will be performed as well.</p> 

{% for post in site.research %}
  {% include archive-single.html %}
{% endfor %}
